### PR TITLE
Version export iOS

### PR DESCRIPTION
**Pull request by RakshithaAcharya**

### CODE REVIEW

**Comment by arshiya-moengage** [comment]
Please add the description and jira link in the description section

File: Core-SDK/Assets/MoEngage/Editor/PostProcessBuild_iOS.cs [hunk 1]
```diff
@@ -140,8 +140,21 @@ public static void OnPostProcessBuild(BuildTarget target, string path)
         
         RemoveExtensionFilesFromMainTarget(path);
 
+        ExportPluginVersion(path);
     }
     
+    private static void ExportPluginVersion(string path) {
+
+        using (StreamWriter sw = File.AppendText(path + "/Libraries/MoEngage/Plugins/iOS/MoEUnityConstants.m"))
```
Comment (arshiya-moengage) [inline Core-SDK/Assets/MoEngage/Editor/PostProcessBuild_iOS.cs]: Why cant we pass the version in the `Initialize()` payload instead?

Comment (RakshithaAcharya) [reply Core-SDK/Assets/MoEngage/Editor/PostProcessBuild_iOS.cs]: @arshiya-moengage Before `initialise` method is called, sdk will already be initialised from native iOS side where integration type and version will be set

**Review by umangmoe — CHANGES_REQUESTED** [review]
This PR should only have version tracking change and not SDK version upgrade. Please raise a separate PR for SDK version upgrade.

**Review by umangmoe — APPROVED** [review]

**Pull request merged**